lib/inspec/resources/http.rb

```diff
@@ -162,98 +164,139 @@ class Remote < Base
         attr_reader :inspec
 
         def initialize(inspec, http_method, url, opts)
-          unless inspec.command("curl").exist?
-            raise Inspec::Exceptions::ResourceSkipped,
-                  "curl is not available on the target machine"
+          if inspec.os.windows?
```

Comment:
The repeated code could be reduced a bit by having

```
          http_cmd = inspec.os.windows? ? 'Invoke-WebRequest' : 'curl'
          unless inspec.command(http_cmd).exist?
            raise Inspec::Exceptions::ResourceSkipped,
                  "#{http_cmd} is not available on the target machine"
          end
```

We _might_ be OK not checking for Invoke-WebRequest since there is a minimum supported PowerShell version (5, I _think_) but it seems wise (though maybe a little slower) to always test.
I also note that in PowerShell you should have `curl` available as an alias to `Invoke-WebRequest` - but I'm not sure this is guaranteed so it might also be wisest to use `Invoke-WebRequest`